interactive_test.py

```diff
@@ -0,0 +1,54 @@
+import os
+import tempfile
+import time
+
+from bluesky import RunEngine
+from bluesky.plans import count
+from databroker import temp
+from ophyd import Component
+from ophyd.areadetector import SimDetector, HDF5Plugin, SingleTrigger
+from ophyd.areadetector.filestore_mixins import (
+    FileStoreHDF5,
+    FileStoreIterativeWrite,
+)
+
+RE = RunEngine()
+db = temp()
+catalog = db.v2
+RE.subscribe(db.insert)
+
+PV_PREFIX = "13SIM1:"
+# prefixes = ['13SIM1:', 'XF:31IDA-BI{Cam:Tbl}']
+
+# Create a directory for the HDF5 file which we will cleanup at the end.
```

Comment:
Where's the cleanup?

Reply:
This file was accidentally committed. Removed and force-pushed.